src/navigation/state.py

```diff
@@ -71,3 +75,29 @@ def evaluate(self, ud):
         cmd_vel = Twist()
         self.context.rover.send_drive_command(cmd_vel)
         return DoneStateTransitions.idle.name  # type: ignore
+
+
+class OffStateTransitions(Enum):
+    _settings_ = NoAlias
+
+    idle = "OffState"
+    begin_course = "WaypointState"
+
+
+class OffState(BaseState):
+    def __init__(self, context: Context):
+        super().__init__(
+            context,
+            add_outcomes=[transition.name for transition in OffStateTransitions],  # type: ignore
+        )
+
+    def evaluate(self, ud):
+        # Check if we need to ignore on
+        if self.context.course and (not self.context.course.is_complete()):
+            return OffStateTransitions.begin_course.name  # type: ignore
+
+        # Stop rover
+        cmd_vel = Twist()
+        self.context.rover.send_drive_command(cmd_vel)
```

Comment:
we don't want to continuously send stop commands while in off state, so probably fix this